prov/sockets/src/sock_dom.c

```diff
@@ -40,7 +40,7 @@
 #include "sock.h"
 #include "sock_util.h"
 
-extern const char const sock_dom_name[];
+extern const char sock_dom_name[];
```

Comment:
I had already submitted some in the warnings pull request I had submitted, but I guess I forgot a couple.